<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LOGIN</class>
 <widget class="QDialog" name="LOGIN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>101</y>
     <width>248</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_nom">
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_username"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_password">
      <property name="text">
       <string>MOTS DE PASSE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="label_verification">
      <property name="text">
       <string>VERIFIER VOTRE NOM ET MOTS DE PASSE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
